content/400-reference/400-utility-types-reference.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Utility types reference
+description: A utility types available for use on the Prisma namespace
+---
+
+<TopBlock>
+
+Utility types are helper functions and types that live on the Prisma namespace. They are useful for keeping your application type safe.
+
+</TopBlock>
+
+## [<inlinecode>Prisma validator</inlinecode>](../concepts/components/prisma-client/working-with-generated-types/#prisma-validator)
```

Comment:
```suggestion
## [<inlinecode>Prisma.validator</inlinecode>](../concepts/components/prisma-client/working-with-generated-types/#prisma-validator)
```

Reply:
or maybe if all the types on this page will have the `Prisma.` prefix we can just omit it?

```suggestion
## [<inlinecode>validator</inlinecode>](../concepts/components/prisma-client/working-with-generated-types/#prisma-validator)
```